### PR TITLE
yaml-cpp: init at 0.6.2

### PR DESCRIPTION
**Pull request by spacekookie**
###### Things done

I did test using this library in a project, which will be packaged at a later time.

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/libraries/yaml-cpp/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    repo = pname;
+    rev = "${pname}-${version}";
+    sha256 = "16lclpa487yghf9019wymj419wkyx4795wv9q7539hhimajw9kpb";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  enableParallelBuilding = true;
+
+  meta = with stdenv.lib; {
```
Comment (alexarice) [inline pkgs/development/libraries/yaml-cpp/default.nix]: Is there no license?

**Review by alexarice — COMMENTED** [review]
Tested with nix review

**Review by veprbl — CHANGES_REQUESTED** [review]
Duplicate of https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/libraries/libyaml-cpp/default.nix

**Comment by spacekookie** [comment]
Oof, I didn't see someone had already packaged it 😅

**Pull request closed without merging**